wandb/internal/update.py

```diff
@@ -36,30 +38,41 @@ def _find_available(current_version):
         release_list = sorted(release_list)
         if not release_list:
             return
+        # Each release is mapped to an array of metadata objects.
+        # Check the "yanked" property for each of the objects.
+        if parsed_current_version in release_list:
+            for item in data["releases"][current_version]:
+                yanked = yanked or item["yanked"]
+        else:
+            deleted = True
+
         parsed_latest_version = release_list[-1]
         if parsed_latest_version <= parsed_current_version:
             return
         latest_version = str(parsed_latest_version)
         pip_prerelease = True
 
-    return (latest_version, pip_prerelease)
+    return (latest_version, pip_prerelease, yanked, deleted)
 
 
 def check_available(current_version):
     package_info = _find_available(current_version)
     if not package_info:
         return
 
-    latest_version, pip_prerelease = package_info
+    latest_version, pip_prerelease, yanked, deleted = package_info
```

Comment:
oh also we're not doing anything w/ `deleted` here lawl. prob should modify `yank_message` if it's `deleted`.

Reply:
I included this per Jeff's request, we wanted to keep track if package was deleted.
In a future PR, I think we're planning to use `deleted` as the sign to kill the script

Reply:
Yeah, I think it'll be super trivial to change the `yank_message` to say "deleted" (maybe with a stronger warning attached) if it's deleted instead of yanked tho.

Reply:
True, I can make that change